toolchain/check/testdata/interface/no_prelude/fail_add_member_outside_definition.carbon

```diff
@@ -79,13 +79,17 @@ interface Outer {
 // CHECK:STDOUT:   witness = ()
 // CHECK:STDOUT: }
 // CHECK:STDOUT:
-// CHECK:STDOUT: interface @Inner
-// CHECK:STDOUT:     generic [@Outer.%Self: %.3] {
-// CHECK:STDOUT:   %Self: %.4 = bind_symbolic_name Self 1 [symbolic = %Self (constants.%Self.3)]
+// CHECK:STDOUT: generic @Inner(@Outer.%Self: %.3) {
+// CHECK:STDOUT: !definition:
+// CHECK:STDOUT:   %Self.2: %.4 = bind_symbolic_name Self 1 [symbolic = %Self.2 (constants.%Self.3)]
+// CHECK:STDOUT: }
+// CHECK:STDOUT:
+// CHECK:STDOUT: interface @Inner {
```

Comment:
Formatting-wise, these look disconnected to me. Is that intentional? Perhaps the `interface` could be moved into the generic at an indent, or add some other connection other than the `@Inner` name match?

Reply:
Done.